gamification/src/main/java/juja/microservices/gamification/user/ServiceImpl.java

```diff
@@ -46,7 +44,7 @@
 public class ServiceImpl implements Service {
```

Comment:
@sLisnychyi it is not good practice to use Imp for implementation. At least if you don't know what for is this class (good question. Do we really need this Service? What logic is encapsulated here ? ) - use Default.